google-cloud-storage/samples/storage_upload_file.rb

```diff
@@ -26,11 +26,11 @@ def upload_file bucket_name:, local_file_path:, file_name: nil
   require "google/cloud/storage"
 
   storage = Google::Cloud::Storage.new
-  bucket  = storage.bucket bucket_name
+  bucket  = storage.bucket bucket_name, skip_lookup: true
 
   file = bucket.create_file local_file_path, file_name
 
-  puts "Uploaded #{local_file_path} as #{file.name} in bucket #{bucket_name}"
+  puts "Uploaded #{local_file_path} as #{file_name} in bucket #{bucket_name}"
```

Comment:
Please revert this line. Calling `file.name` will not trigger an additional call. And verifying `file.name` is important for testing this sample.